webcomponents.js/webcomponents.js-tests.ts

```diff
@@ -10,10 +10,12 @@ var fooProto = Object.create(HTMLElement.prototype, {
     }
 });
 
-document.registerElement("x-foo", {
+var XFoo = document.registerElement("x-foo", {
     prototype: fooProto
 });
 
+var xFoo = new XFoo();
+
```

Comment:
Hello vvakame, thanks for asking, although I am not one of the mergers.

A few comments:
* I provided AbstractPolymerElement in tests as a convenience for developer who wish to integrate easily, maybe you could provide an AbstractDomModule, and an additional test case to register an instance of a simple implementation, showing the object-oriented way of using Polymer ?
* I started to write the definitions of the core (now Iron) & paper components' library. It would be nice to write at least the one I wrote for 0.5 (except the obsoletes one, only a few)
* I can help if you want :)

Thanks for your work!